app/src/main/java/com/grignon/shanghai/entity/Player.java

```diff
@@ -2,14 +2,24 @@
 
 import androidx.annotation.NonNull;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public class Player {
-    private final String nickname;
+    private String socketId;
+    private String nickname;
```

Comment:
C'est que des **get** donc tu peut les mettre `final`
```suggestion
    private final String socketId;
    private final String nickname;
```